dask/array/core.py

```diff
@@ -225,7 +225,11 @@ def slices_from_chunks(chunks):
     return list(product(*slices))
 
 
-def getem(
```

Comment:
This function certainly seems to be private/internal, but perhaps we should get confirmation that it isn't being used directly by down-stream libraries (like xarray) before removing?

Reply:
A [quick search]( https://github.com/pydata/xarray/search?q=getem&type= ) seems to indicate it is unused in xarray. Are there other projects we should check with as well?

Reply:
In the Dask org there are a couple old blogposts that reference this `getem` function. Though Idk if people are still looking at blogposts that far back or using them anywhere

https://blog.dask.org/2014/12/27/Towards-OOC
https://blog.dask.org/2014/12/30/Towards-OOC-Frontend

Reply:
I didn't remove it, but renamed it `graph_from_arraylike`, as something a bit more descriptive. I'd be happy to roll that back in the interests of backwards compatibility (thought the signature of the function will have also changed to be HLG-ified, so the backwards compatibility ship may have sailed)

Reply:
It's pretty straighforward to add a note to old blogposts telling people things have changed in more recent versions of Dask. We could do that if you want.

Reply:
Hi, this broke Vaex, in case anyone reads this, this was my compatibility fix:

- https://github.com/vaexio/vaex/pull/1873